src/main/java/org/fenixedu/academic/ui/spring/AcademicConfigurationsController.java

```diff
@@ -0,0 +1,11 @@
+package org.fenixedu.academic.ui.spring;
+
+import org.fenixedu.bennu.spring.portal.SpringApplication;
+import org.fenixedu.bennu.spring.portal.SpringFunctionality;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@RequestMapping("/academic-configurations")
+@SpringApplication(group = "logged", path = "academic-configurations", title = "title.academic.configurations")
+@SpringFunctionality(app = AcademicConfigurationsController.class, title = "title.academic.configurations")
+public class AcademicConfigurationsController {
```

Comment:
What is the purpose of this? (I can understand the class being marked as an Application, but not the marking it as a Controller).